app.py

```diff
@@ -27,6 +27,10 @@
 
 
 def run_docopt(doc, argv):
+    try:
+        argv = shlex.split(argv)
```

Comment:
This works but I think it's best to do the splitting elsewhere, so that `run_docopt` can accept a list of arguments, as is usual.